### PR TITLE
TNL-3065 fix

### PR DESCRIPTION
**Pull request by e-kolpakov**
Fix for [TNL-3065](https://openedx.atlassian.net/browse/TNL-3065): poll shows voting options if voted and subsection navigation is used

### CODE REVIEW

**Comment by smarnach** [comment]
This PR should target the edx-release branch first, since this is going to go on edx.org.  We can then merge edx-release back into master, as mentioned in the ticket for this issue.

File: poll/poll.py [hunk 1]
```diff
@@ -876,6 +883,13 @@ def studio_submit(self, data, suffix=''):
 
         return result
 
+    @XBlock.json_handler
+    def student_voted(self, data, suffix=''):
+        return {
+            'voted': self.get_choices() is not None,
+            'private_results': self.private_results
+        }
+
```
Comment (smarnach) [inline poll/poll.py]: Given that the two implementations are identical, can't this be added to `PollBase`?

File: poll/public/js/poll.js [hunk 2]
```diff
@@ -2,28 +2,20 @@
 
 function PollUtil (runtime, element, pollType) {
     var self = this;
+    debugger;
```
Comment (smarnach) [inline poll/public/js/poll.js]: Please remove.

**Comment by smarnach** [comment]
Code looks good, except for the `debuger` invocation.  As pointed out above, I believe this PR should actually target the edx-platform branch, which will later be merged into master.  Since this contains only a single commit, it would also be possible to merge into master first and cherry-pick into edx-release later.

I tested this on my devstack without finding any issues.

**Comment by e-kolpakov** [comment]
Closed in favor of #5

**Pull request closed without merging**